src/main/scala/uk/gov/hmrc/govukfrontend/views/Implicits.scala

```diff
@@ -188,6 +189,32 @@ trait Implicits {
 
     private def errorMessage(formError: FormError) = messages(formError.message, formError.args: _*)
   }
+
+
+  implicit class RichRadios(radios: Radios)(implicit messages: Messages) {
+
+    /**
+      * Extension method to allow a Play form Field to be used to override certain parameters in a Radios,
+      * specifically errorMessage, idPrefix, name, and checked (for a specific RadioItem)
+      *
+      * @param field
+      * @param messages
+      */
+    def withFormField(field: Field): Radios = {
+      radios.copy(
+        errorMessage = field.error.map(formError =>
+          ErrorMessage(content = Text(messages(formError.message, formError.args: _*)))
+        ),
+        idPrefix = Some(field.name),
```

Comment:
As discussed, would we want to override only if idPrefix hasn't already been specified?
i.e.
```
idPrefix = idPrefix.orElse(field.name)
```

Reply:
Updated